roles/dropwizard-api/tasks/gen-certs.yml

```diff
@@ -0,0 +1,39 @@
+---
+  - name: copy gen-certs.sh to dest
+    template:
+      src: gen-certs.j2
+      dest: /apis/gen-certs.sh
+    become_user: osu_apis
+
+  - name: stat gen-certs.sh
+    stat:
+      path: /apis/gen-certs.sh
+    register: gen_certs_stat
```

Comment:
@magical The reason why I added checking condition here is because ansible-lint will give us warning if there is no need to run 'shell' or 'command' module. However, we should always execute this shell script whenever we run the role. This is a compromised way I could figure out so far.